### ISSUE TITLE
Website

### ISSUE THREAD

**Issue by mp4096** [mp4096/indentex#21]
The website should be an entry point for Indentex beginners.

It should have:

- [ ] a pitch
- [ ] an antipitch (cf. [`ripgrep`'s antipitch](https://github.com/BurntSushi/ripgrep/blob/d2c7a76a3c0f6c264ec0b1f04e1dcaeba3862a51/README.md#why-shouldnt-i-use-ripgrep))
- [ ] comparison to other alternatives (`pandoc` etc.)
- [ ] examples for indentex usage
- [ ] best practices (e.g. which parts of project should be done in pure LaTeX and which in indentex)
- [ ] links to binary downloads
- [ ] installation instructions
- [ ] tooling instructions (e.g. how to integrate it into TeXstudio etc.)

**Comment by syxolk** [comment]
**Funny idea:**
Use the `wasm32-unknown-emscripten` target to compile to WebAssembly (or asm.js) and use this in the browser to provide a live transpiler demo. Downside: The emscripten SDK needs 24 GB of space (wtf).

Sources:
- https://hackernoon.com/compiling-rust-to-webassembly-guide-411066a69fde
- https://users.rust-lang.org/t/compiling-to-the-web-with-rust-and-emscripten/7627
- https://blog.rust-lang.org/2016/12/22/Rust-1.14.html

**Comment by mp4096** [comment]
Yeah, a live demo would be cool. However, using wasm/emscripten seems to be unreasonable at the moment, I'd wait a couple of years.

I mean, using Rust for writing client-side code is weird. It's like using JavaScript for server programming. Oh wai...
